automation-scripts.md

```diff
@@ -0,0 +1,195 @@
+- [Reaching Out](#reaching-out)
+- [Disclaimer](#disclaimer)
+- [LICENSE](#license)
+
+## Services
+
+JC Manciot's GitLab-hosted [PPA](https://gitlab.com/jean-christophe-manciot/ppa) offers a collection of scripts for automated drive commands including syncing which can be used on the latest stable Ubuntu distribution.
+They are all included in the **drive-google** package.
+
+The ```drive-<command>``` scripts offer the following services:
+  + **abstraction** from underlying drive program, allowing the user to use it without knowing all its options & details
+  + **automation**, allowing the user to launch each command once on all its selected objects
+  + **synchronization** offering 3 types of operation:
+    - *Fully-Automated*: it does not ask for user confirmation (not implemented yet)
+    - *Semi-Automated*: it asks for user confirmation
+    - *Manual*: it asks for user confirmation & offers a choice of a combination of 8 commands 
+
+    The first 2 operations allow 3 different types of **master**:
+    - *None*: all local & remote modified objects are synced, but trashed/deleted objects are not synced
+    - *Local*: your remote Google Drive is mirrored from your local GD
+    - *Remote*: your local Google Drive is mirrored from your remote GD
+  + **choice of user interface**:
+    - *CLI commands*: all scripts begin with ```# drive-<command>```
+    - *GUI menus*: all previous commands are accessible through one command in basic graphical mode, ```# drive-menu```
+  + results in basic **graphical windows**
+  + **agentless**: no daemon is running on your local host
+
+## Installing The Automation Scripts
+
+### Stable Channel
+
+Installing the scripts through a **stable channel** where <latest_release_code_name> below is the first word only of the latest Ubuntu release (yakkety for example):
+```sh
+# sudo sh -c $'echo "deb https://gitlab.com/jean-christophe-manciot/ppa/raw/master/Ubuntu <latest_release_code_name> stable #JC Manciot\'s Stable PPA" >> /etc/apt/sources.list.d/jean-christophe-manciot.list'
+# sudo apt-get update
+# sudo apt-get install drive-google
+```
+
+### Unstable Channel
+
+Installing the scripts through an **unstable channel** where <latest_release_code_name> below is the first word only of the latest Ubuntu release (yakkety for example):
+```sh
+sudo sh -c $'echo "deb https://gitlab.com/jean-christophe-manciot/ppa/raw/master/Ubuntu <latest_release_code_name> unstable #JC Manciot\'s Unstable PPA" >> /etc/apt/sources.list.d/jean-christophe-manciot.list'
+```
+
+## Initializing Drive
+```sh
+# mkdir GD
+# cd GD
+GD# drive init
+Visit this URL to get an authorization code
+https://accounts.google.com/o/oauth2/auth?access_type=offline&client_id=354790962074-7rrlnuanmamgg1i4feed12dpuq871bvd.apps.googleusercontent.com&redirect_uri=urn%3Aietf%3Awg%3Aoauth%3A2.0%3Aoob&response_type=code&scope=https%3A%2F%2Fwww.googleapis.com%2Fauth%2Fdrive&state=2017-01-10+08%3A03%3A00.659412311+%2B0100+CET2596996162
```

Comment:
You probably don't want to hardcode this URL in here, in the README. Maybe in the gitlab repo but not here, doesn't look good security wise, even if in code we have hard-coded credentials.